docs/howto/delete-app.txt

```diff
@@ -0,0 +1,32 @@
+==========================
+How to delete a Django app
+==========================
+
+To delete an app from you project you need to follow the next steps:
+
+1. Remove all references to the app (i.e imports).
+
+2. Run ``python manage.py migrate <app_label> zero``.
+
+3. Remove your app from :setting:`INSTALLED_APPS` and delete the directory.
+
+4. Run :djadmin:`remove_stale_contenttypes` command passing
+   ``--include-stale-apps``.
+
+.. warning::
+
+    It could happen in the future you want delete other app and it has
+    dependencies from the app you previously removed and get an error like:
+
+    .. code-block:: shell
+
+        django.db.migrations.exceptions.NodeNotFoundError: Migration
+        <app_label_to_delete>.XXXX_migration_example dependencies reference
+        nonexistent parent node ('<app_label_deleted>', 'XXXX_migration_example')
```

Comment:
I created a repo to exemplify this case: https://github.com/mgaligniana/ticket_34309/commits/main

1. Clone the project
2. `python -m venv env`
3. `source env/bin/activate`
4. `pip install django`
5. Try do the first step to delete Books app doing `python manage.py migrate books zero`